src/prelaunch/cannonball.bltz.a

```diff
@@ -18,6 +18,12 @@
          sta   $7dbf      ; patch - don't increase lives
          sta   $9c01      ; patch - don't decrease lives
 +
+         +RESET_VECTOR $100
```

Comment:
Do we need to do this? TR will set this before calling a prelaunch file. Does the game try to change it before this?

Reply:
Yes it does. I'm making the assumption that if the decompress doesn't change anything, we don't need to touch it. Therefore, anytime we are re-setting it is because it was changed before that point.

Reply:
Also, I try to change it as close to when the game sets its own vector, as well. Sometimes that's in part1 or sometimes it's later. That's why it can appear in different sequences.